src/main/java/net/darksky/project/numberone/CreateForecast.java

```diff
@@ -0,0 +1,52 @@
+package net.darksky.project.numberone;
+
+import java.io.IOException;
+
+import tk.plogitech.darksky.forecast.APIKey;
+import tk.plogitech.darksky.forecast.ForecastRequest;
+import tk.plogitech.darksky.forecast.ForecastRequestBuilder;
+import tk.plogitech.darksky.forecast.GeoCoordinates;
+import tk.plogitech.darksky.forecast.model.Forecast;
+import tk.plogitech.darksky.forecast.model.Latitude;
+import tk.plogitech.darksky.forecast.model.Longitude;
+
+public class CreateForecast {
```

Comment:
This class doesn't create anything. It is a model and should be named accordingly. 
But IMO you don't need it. The thing is fields of this class are not connected by behavior.
For instance `apiKey` is something that will always remain the same when `forecast` will change depending on the result from the server.
`latitude` and `longitude` are something user enters when `forecast` is a result from the API.
This kind of things of different nature should not be mixed up in one class.